schema/content-descriptor.json

```diff
@@ -23,6 +23,10 @@
         "type": "string",
         "format": "uri"
       }
+    },
+    "annotations": {
+      "id": "https://opencontainers.org/schema/image/descriptor/annotations",
+      "$ref": "defs-image.json#/definitions/annotations"
```

Comment:
This catches us up with #501 (and it needs an `fs.go` bump too).  With the spec change already hashed out in #501, this can probably get spun off into its own PR and landed faster than the index pivot as a whole.